include/deal.II/lac/precondition.h

```diff
@@ -986,7 +1047,13 @@ class PreconditionChebyshev : public Subscriptor
   mutable VectorType update2;
 
   /**
-   * Stores the additional data provided to the initialize function.
+   * Internal vector used for the <tt>vmult</tt> operation.
+   */
+  mutable VectorType update3;
```

Comment:
You probably want to guard this mutable variable with a mutex, to make sure that one can call all `const` function more than once in parallel. I don't know whether that's going to be a common use case, but it's so incredibly hard to find these kinds of bugs that I think we do well to *always* guard mutable variables with a mutex.